File: TestToolbar/app/src/main/java/com/pix/testtoolbar/MainActivity.java
package com.pix.testtoolbar;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.support.v7.widget.Toolbar;
import android.view.View;

public class MainActivity extends AppCompatActivity {
    private Toolbar toolbar;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        initView();
    }

    private void initView() {
        toolbar = (Toolbar) findViewById(R.id.id_toolbar);
        toolbar.setTitle("TestToolbar");
        setSupportActionBar(toolbar);
    }

    public void demo01Click(View v) {
        startActivity(new Intent(this,Demo01Activity.class));
    }
}

File: TestDialogFragment/app/src/main/java/com/pix/testdialogfragment/MainActivity.kt
package com.pix.testdialogfragment

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    var simpleDialogFragment: SimpleDialogFragment? = null
    var customViewDialogFragment: CustomViewDialogFragment? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initView()
    }

    private fun initView() {
        simpleDialogFragment = SimpleDialogFragment()
        btn_simple_dialog.setOnClickListener {
           simpleDialogFragment?.show(supportFragmentManager, "simple")
        }
        customViewDialogFragment = CustomViewDialogFragment()
        btn_custom_view_dialog.setOnClickListener {
            customViewDialogFragment?.show(supportFragmentManager, "custom view dialog")
        }
    }

}